website/project/metadata/aea-prereg-2.json

```diff
@@ -0,0 +1,480 @@
+{
+    "name": "AEA_Preregistration",
+    "version": 2,
+    "description": 
+    "pages": [{
+        "id": "page1",
+        "title": "",
+        "contents": [{
+            "id": "general",
+            "type": "section",
+            "contents": [{
+                "required": true,
+                "type": "textfield",
+                "id": "title",
+                "label": "Trial Title",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "registration_number",
+                "label": "AEA RCT Registration Number",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "registration_date",
+                "label": "Registration Date",
+                "caption": "caption"
+            }, {
+                "type": "textfield",
+                "id": "updated",
+                "label": "Last Updated",
+                "caption": "caption"
+            }],
+            "title": "General Information"
+        }, {
+            "type": "section",
+            "canRepeat": true,
+            "id": "location",
+            "contents": [{
+                "type": "select",
+                "required": true,
+                "options": [
```

Comment:
Pinging @lbanner as last person we can find to have looked at this.

This schema seems to be malformed; the list of countries seems truncated in a way that suggests human error (places that start with A, then W-Z). Two questions:

1. Do we intend the AEA registrations to be part of this PR? **If not, it would make sense to remove this file as being out of scope of this PR**. (rather than let a malformed but unused file slip through the cracks)

2. When we do activate the AEA feature, we should be sure to validate the list of countries and correct any schema errors.